encord/orm/label_row.py

```diff
@@ -222,6 +253,7 @@ class LabelRowMetadata(Formatter):
     data_type: str
     label_status: LabelStatus
     annotation_task_status: AnnotationTaskStatus
+    export_history: List[LabelRowExportHistory] = field(default_factory=list)
```

Comment:
I don't think the `field` is needed here as we should always be populating this one right? 

Not sure why this defensive programming is needed here.